optimum/intel/ipex/inference.py

```diff
@@ -1,11 +1,22 @@
+import inspect
 import logging
-from typing import Union
+import re
+from dataclasses import dataclass
+from typing import Optional, Tuple, Union
 
 import torch
 from torch import nn
-from transformers import add_start_docstrings
+from transformers import GenerationMixin, PreTrainedModel, add_start_docstrings
+from transformers.modeling_outputs import CausalLMOutputWithPast, Seq2SeqLMOutput
 from transformers.pipelines import Pipeline
-from transformers.utils import is_ipex_available
+from transformers.utils import ModelOutput, is_ipex_available
+
+from optimum.exporters.onnx import OnnxConfig
+from optimum.exporters.tasks import TasksManager
+from optimum.utils import DummyPastKeyValuesGenerator, NormalizedConfigManager, NormalizedTextConfig
+
+from ..openvino import _TASK_ALIASES

```

Comment:
This could cause issue as openvino is not a hard dependecy, you can rebase and use :

```suggestion
from ..utils.constant import _TASK_ALIASES
```
(moved in https://github.com/huggingface/optimum-intel/pull/266)